src/components/stepThree/index.js

```diff
@@ -364,7 +360,7 @@ export class stepThree extends React.Component {
             <div className="hidden">
               <div className="input-block-container">
                 <InputField
-                  side="left"
+                  extraClassName="left"
```

Comment:
it should be `side` for the `InputField`

Reply:
Oops, thanks. Fixed.